github-diff-filename.user.js

```diff
@@ -23,6 +23,15 @@
 	const arrow = "\u2192"; // "→"
 	const regex = new RegExp(`\\s${arrow}\\s`);
 
+    function createStyleSheet(green, red){
+        let style = document.createElement('style');
+        style.innerHTML = `
+        .ghdfn.text-green{ color: ${green}; }
+        .ghdfn.text-red{ color: ${red}; }
```

Comment:
Looks like it was based on GitHub's classes and they changed it ;-)

![image](https://user-images.githubusercontent.com/1692136/143781747-17bd5275-89e1-431b-bac6-ce12bab6cb26.png)
![image](https://user-images.githubusercontent.com/1692136/143781739-ed48f271-bb8a-435c-9371-79380cb73b80.png)

Maybe you should just update them to stick with GitHub's colors? :beers: